katas/chicken-sexing/mateo/chickenSexing.js

```diff
@@ -1,3 +1,14 @@
 export function correctness(bobsDecisions, expertDecisions) {
+    let points = 0;
+    for (let i = 0; i < bobsDecisions.length; i++){
```

Comment:
Please add space between `)` and `{` in order to keep consistency, same in the lines below